contribute.md

```diff
@@ -19,13 +19,18 @@ Pull requests should be made targeting the `master` branch. Every few weeks, the
 
 You can run the site locally on your computer while making changes.
 
+### Setup Ruby and Bundler
+
 Ensure that you have Ruby and [Bundler](http://bundler.io/) installed.
 
+    brew install ruby
```

Comment:
This assumes Mac OS which is why I didn't include it. I think it's safe to skip instructions for installing Ruby.

Reply:
removed